recipe/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "peft" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/huggingface/peft/archive/v{{ version }}.tar.gz
+  sha256: 954b86dcf3dad64562ce7b9167485ac42c47fcc668d82ea99874001aa585f53e
+
+build:
+  skip: True  # [s390x or ppc64le]
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
+
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - numpy >=1.17
+    - packaging >=20.0
+    - psutil
+    - pyyaml
+    - pytorch >=1.13.0
+    - transformers
+    - tqdm
+    - huggingface_accelerate >=0.21.0
+    - safetensors
+
+test:
+  source_files:
+    - tests
+  imports:
+    - peft
+  commands:
+    - pip check
+    # skip test_prompt_encoder_warning_num_layers, see https://github.com/huggingface/peft/issues/1066
+    - pytest tests -k "not test_prompt_encoder_warning_num_layers"  # [not win]
+    # skip failing windows tests
+    - pytest tests -k "not (test_prompt_encoder_warning_num_layers or test_feature_extraction_models.py or test_decoder_models.py or test_encoder_decoder_models.py)"  # [win]
+  requires:
+    - pip
+    - parameterized
+    - datasets
+    - diffusers <0.21.0
+    - pytest
```

```suggestion
    - pytest
    - pytest-xdist
```
See https://github.com/huggingface/peft/blob/4df9c5a243194b03e703c1dd526d64163f9b4fd2/setup.py#L21C60-L21C72